### ISSUE TITLE
innerText

### ISSUE THREAD

**Issue by christianbundy** [1N50MN14/html-element#43]
Hi! I have some HTML like:

```html
<p><strong>Hello</strong> world</p>
```

And I want:

```
Hello world
```

Usually I'd use [innerText](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/innerText), but it doesn't seem to be available here. Is there another way of getting the text that a browser would render? Thanks!

**Comment by tdumitrescu** [comment]
How about `textContent`? Assuming you don't need the fine-grained display awareness of `innerText` but just a blunt instrument to grab text like in your example... https://github.com/1N50MN14/html-element/blob/b8a15b3335eb2bde8d74849c8cd63bea608714b7/index.js#L346-L359

**Comment by christianbundy** [comment]
Oh, thanks for the tip. I tried it but received empty string, but I think that's a bug? Am I doing something wrong here?

```javascript
var document = require('html-element').document;
var div = document.createElement('div')
div.innerHTML = '<p><strong>Hello</strong> world</p>'
div.textContent // => ''
```

**Comment by tdumitrescu** [comment]
Yeah, it's an anemic innerHTML implementation: https://github.com/1N50MN14/html-element/blob/b8a15b3335eb2bde8d74849c8cd63bea608714b7/index.js#L260-L267

AFAICT right now with this lib you'd need to create the elements explicitly like with `document.createElement()`.
